### ISSUE TITLE
💡 New Rule: createFeatureSelector should only be used with 1 generic for the feature state

### ISSUE THREAD

**Issue by timdeschryver** [timdeschryver/eslint-plugin-ngrx#155]
## Information about the new rule

https://github.com/ngrx/platform/issues/2849#issuecomment-752717086

## A snippet of incorrect code

```ts
const featureA = createFeatureSelector<State, FeatureAState>('featureA');
```

## A snippet of correct code

```ts
const featureA = createFeatureSelector<FeatureAState>('featureA');
```

## I would be willing to submit a PR for the docs :heart:

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

**Comment by ValentinBossi** [comment]
happy to see the "Rule Details" part! :)
maybe some sentences about lazy loading?

**Comment by timdeschryver** [comment]
Closed by #157
